custom-api/create_data.py

```diff
@@ -0,0 +1,135 @@
+import os
+
+# Databricks SDK
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.service.sql import *
+
+# Initialize Databricks SDK. 
+# By default, this authenticates using the DATABRICKS_HOST and DATABRICKS_TOKEN environment variables
+# initialized by setup.sh.
+w = WorkspaceClient()
+
+warehouse_id = os.environ.get("DATABRICKS_WAREHOUSE_ID")
+
+# This script uses the Statement Execution API to generate sample data.
+
+def execute_statement_sync(sql_statement):
+  response = w.statement_execution.execute_statement(
+    statement = sql_statement,
+    warehouse_id = warehouse_id
+  )
+
+  while response.status.state in (StatementState.PENDING, StatementState.RUNNING):
+    response = w.statement_execution.get_statement(statement_id = response.statement_id)
+
+  if response.status.state != StatementState.SUCCEEDED:
+    print("Failed to execute: '" + sql_statement + "' with state " + str(response.status.state));
+    if response.status.state == StatementState.FAILED:
+        print("Detailed error message: " + response.status.error.message)
+    exit()
+
+  return
+
+# Create a database.
+print("Creating acme_demo database.")
```

Comment:
Should we check if it already exists and bail out if so?
Currently it fails with an error and you are not sure if the rest of the setup is done.